src/data/hassio/common.ts

```diff
@@ -21,12 +21,20 @@ export interface HassioStats {
 export const hassioApiResultExtractor = <T>(response: HassioResponse<T>) =>
   response.data;
 
-export const extractApiErrorMessage = (error: any): string =>
-  typeof error === "object"
+export const extractApiErrorMessage = (
```

Comment:
This is code from the supervisor, it lives in `data/hassio`, if we want to make it more generic we should move it out of the code for supervisor.